Voog/design-amsterdam
# Add sale price support

Adds sale price support, which includes:

- Displaying sale prices to auto-generated product pages
- Displaying sale prices to product list layout
- Making sale prices react to dark backgrounds
- Making sale prices react to design editor changes

In addition, fix colour variable names in product list layout.

Closes #169 

Testing:

- Make sure that sale prices are displayed on auto-rendered product pages & product list layouts
- Make sure that sale prices react to design editor changes
- Make sure that sale prices react to dark background on `Feature page` layout
